### DISCUSSION TITLE
email address

### DISCUSSION THREAD

**Discussion by TeddygV** [community/community#33411] [Actions]
Hello there,

I am trying to add a school email so that I can gain access to the pro version of github for school, I keep trying to send the verification email and unfortunately it doesn't seem to work. I just sends me back to the account page. Is there anyone that can help me with this?

Thank you,

TeddyV

**Comment by ChrisioGwaan** [comment]
Hi,
I experienced the same before. It seems that your school account has email delayed, including checking size of the email and its attachments, network latency, spam/virus etc.
This link might help you fix the issue.
https://answers.microsoft.com/en-us/outlook_com/forum/all/office-emails-delayed-school-account/e30dd39b-a388-4e94-bfed-c9985acc6a65
